src/main.rs

```diff
@@ -134,6 +148,15 @@ const ARGUMENTS_TO_REMOVE: &[&str] = &[
 ];
 
 fn main() {
+    #[cfg(feature = "sentry")]
```

Comment:
Could this be combined with the next lines to a single block, to make it a bit cleaner?